### PR TITLE
Add tab atom to dress code

### LINKED ISSUE #238: Add vertical/horizontal tab to dress code
Sometimes a number of charts or lists are to be shown to user, but user doesn't view them simultaneously. These view may or may not share context.

Dress code should have vertical/horizontal tab css to show tab component as below.

screen shot 2016-08-15 at 18 04 45

screen shot 2016-08-15 at 18 04 55

screen shot 2016-08-15 at 18 05 13

### PR DESCRIPTION
**Pull request by ashwinaggarwal**
Closes #238 

@gabrielhl I have made the changes as per the input on the issue, please review.

screen shot 2017-04-16 at 17 18 48

### CODE REVIEW

**Comment by rbarilani** [comment]
@ashwinaggarwal can you update your branch with the latest master, in theory will fix the build.

File: src/styles/atoms/_tab.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    $tab-active-selector: dc-tab__element--active
+) {
+
+    &.#{$tab-active-selector} {
+        @include dc-tab__element--vertical--active;
+    }
+}
+
+@mixin dc-tab__element--active {
+    color: $dc-orange40;
```
Comment (rbarilani) [inline src/styles/atoms/_tab.scss]: @ashwinaggarwal Can we think about a way to have the color for tab customizable?

Something like:

```scss
// _variables.scss
$dc-tab-color: $dc-orange-40;
```

```scss
// _tabs.scss
// example for dc-tab__element--active mixin
@mixin dc-tab__element--active($color: $dc-tab-color) {
   color: $color;
}
// .... and so on for the other mixins?
```

@ashwinaggarwal WDYT?

Comment (ashwinaggarwal) [inline src/styles/atoms/_tab.scss]: It will be better definitely. I will make the changes.

Comment (rbarilani) [inline src/styles/atoms/_tab.scss]: @ashwinaggarwal any news?

**Comment by cjhowald** [comment]
What's the status of this PR? Is it waiting for changes, or waiting for review? @ashwinaggarwal @rbarilani

**Comment by rbarilani** [comment]
@cjhowald there is just one little change requested... but we can do it directly after merge... BTW LGTM.

**Comment by rbarilani** [comment]
👍

**Comment by ashwinaggarwal** [comment]
@rbarilani @cjhowald moving default tab color to variables was good idea, I have moved it now!

**Comment by rbarilani** [comment]
👍

**Comment by cjhowald** [comment]
👍

**Pull request merged**